### ISSUE TITLE
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory

### ISSUE THREAD

**Issue by okonet** [webpack/webpack-dev-server#215]
When compiling JS or CSS resources using webpack-dev-server, I'm constantly getting `FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory` after a few compiles and need to restart the server manually.

node v0.12.7
npm 2.11.3
webpack 1.10.1

**Comment by Gabriel0402** [comment]
+1

**Comment by SpaceK33z** [comment]
Do you still have this answer? If so, do you happen to use `[hash]` in your filenames?

**Comment by SpaceK33z** [comment]
I'm closing this issue because of inactivity. Feel free to comment, and we can always re-open it again.

**Comment by webdevnerdstuff** [comment]
I was having this issue crashing the dev server all the time. I took what I found someplace else being applied to webpack.cmd (which didn't work for me) and applied it to webpack-dev-server.cmd and I *think it's working better now. Hasn't crashed since yesterday when I changed it so far (knock on wood). Update the webpage-dev-server.cmd file to use `--max_old_space_size=4096`

**File:**
node_modules > .bin > webpack-dev-server.cmd

**Updated to:**
```
@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe" --max_old_space_size=4096  "%~dp0\..\webpack-dev-server\bin\webpack-dev-server.js" %*
) ELSE (
  @SETLOCAL
  @SET PATHEXT=%PATHEXT:;.JS;=;%
  node --max_old_space_size=4096 "%~dp0\..\webpack-dev-server\bin\webpack-dev-server.js" %*
)
```
